examples/index.md

```diff
@@ -389,127 +342,61 @@ Content-Type: application/vnd.api+json
-
-```http
-GET /api/posts/1
 ```
-
-```http
-HTTP/1.1 400 Bad Request
-Content-Type: application/vnd.api+json
-
 {
-  "errors": [
-    {
-      "title":  "Missing Query Parameter",
-      "source": { "parameter": "auth_key" },
-      "detail": "The required parameter, auth_key, is missing."
-    }
-  ]
+  "errors": [{
+    "status": "400",
+    "detail": "JSON parse error - Expecting property name at line 1 column 2 (char 1)."
+  }]
 }
 ```
 
-Notice that the `source` indicates the URI query parameter which caused the error.
-This example with a missing `auth_key` parameter was chosen over an example with an invalid
-URI query parameter, since it may make sense to ignore invalid parameters in some cases, and
-have the request succeed rather than fail. As described in the [Bulk
-extension](http://jsonapi.org/extensions/bulk/#bulk-operations):
-
-> ...a request MUST completely succeed or fail (in a single "transaction").
-> ...any request that involves multiple operations MUST only succeed if all operations are performed
-successfully
-
-Examples of invalid parameters may be `?page[prev]=` (parameter present, but has no value),
-`?include=auther` (invalid parameter value, perhaps a typo), `?felds[author]=`,
-(invalid parameter key, perhaps a typo), `?redirect_to=http%3A%2F%2Fwww.owasp.org` (invalid
-parameter, in this case, a phishing attack), etc.
-
-### Using JSON API extensions
-
-Request and response using the [JSON PATCH
-Extension](http://jsonapi.org/extensions/jsonpatch/):
-
-```http
-PATCH /posts/1
-Content-Type: application/vnd.api+json; ext=jsonpatch
-Accept: application/vnd.api+json; ext=jsonpatch
-
-[ { "op": "remove" } ]
-```
-
-```http
-HTTP/1.1 422 Unprocesssable Entity
-Content-Type: application/vnd.api+json; ext=jsonpatch
-
-[
-  {
-    "errors": [
-      {
-        "status": "422",
-        "detail": "Operation missing required keys: 'path'"
-      }
-    ]
-  }
-]
-```
-
-Notice that JSON PATCH is a special semantics and does not mean an HTTP PATCH request
-containing JSON.
+#### Invalid Query Parameters
 
-Request and response using the [Bulk extension](http://jsonapi.org/extensions/bulk/)
+The `source` member can also be used to indicate that the error originated
+from a problem with a URI query parameter, like so:
 
 ```http
-PATCH /photos
-Content-Type: application/vnd.api+json; ext=bulk
-Accept: application/vnd.api+json; ext=bulk
-
-{
-  "data": [{
-    "type": "articles",
-    "id": "1",
-    "attributes": {
-      "title": "To TDD or Not"
-    }
-  }, {
-    "type": "articles",
-    "id": "2",
-    "attributes": {
-      "title": "LOL Engineering"
-    }
-  }]
-}
+GET /api/posts/1?include=auther
 ```
 
 ```http
 HTTP/1.1 400 Bad Request
 Content-Type: application/vnd.api+json
 
-
 {
   "errors": [
     {
-      "status": "404",
-      "source": { "pointer": "/data/articles/1" },
-      "detail": "Article with id 1 could not be found."
+      "source": { "parameter": "include" },
```

Comment:
I switched this to using an invalid `?include` value because the server cannot, in fact, just ignore such an error:

> If a server is unable to identify a relationship path or does not support inclusion of resources from a path, it MUST respond with 400 Bad Request.